lasy/utils/grid.py

```diff
@@ -173,9 +173,11 @@ def temporal2spectral_fft(self):
         (Only along the time axis, not along the transverse spatial coordinates.)
         """
         assert self.temporal_field_valid
+
         self.spectral_field = np.fft.ifft(
-            self.temporal_field, axis=time_axis_indx, norm="backward"
+            np.fft.fftshift(self.temporal_field, axes=-1), axis=-1
         )
+
```

Comment:
I think it's better if collapse it into two lines to make it easier to read. 
```suggestion
    shifted_temporal = np.fft.fftshift(self.temporal_field, axes=-1) 
    self.spectral_field = np.fft.ifft(shifted_temporal, axis=-1)
```